contracts/StakeManagerV2.sol

```diff
@@ -213,7 +245,7 @@ contract StakeManagerV2 is
     }
 
     /// @notice Complete the undelegation process.
-    function completeBatchUndelegation() external override nonReentrant whenNotPaused {
+    function completeBatchUndelegation() external override nonReentrant {
```

Comment:
for a safe side, add access restriction on `completeBatchDelegation` or else it must error `outOfBounds`